src/app/components/copy-to-clipboard/index.html

```diff
@@ -0,0 +1,45 @@
+<sky-demo-page
+  pageTitle="Copy to clipboard"
+  skyuxModule="SkyClipboardModule"
+  packageName="lib-clipboard"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The copy to clipboard component creates a button that enables users to copy content to their clipboard.
+    </p>
+    <p>
+      <sky-alert alertType="info">
+        To use the copy to clipboard component, set it up as a dependency in your SKY UX SPA. First, install the package as part of your SPA dependencies: <stache-code>npm install @blackbaud/skyux-lib-clipboard --save</stache-code>. Then add <stache-code>SkyClipboardModule</stache-code> as both an import and export in your <stache-code>app-extras.module.ts</stache-code> file.
+      </sky-alert>
+    </p>
+  </sky-demo-page-summary>
+
+  <sky-demo-page-properties>
+    <sky-demo-page-property
+      propertyName="copyTarget"
+      isOptional="true"
```

Comment:
Is this indeed optional? How does the directive know what to copy if you don't specify this?

Reply:
Oops, you're correct; this isn't optional. Fixing!

Reply:
I went ahead and removed `isOptional` and also reorganized the other 2 properties to follow alphabetical order. I'll push that shortly, but I'll go ahead and resolve the comment.